mmv1/templates/terraform/examples/scc_v2_organization_mute_config_basic.tf.erb

```diff
@@ -0,0 +1,8 @@
+resource "google_scc_v2_organization_mute_config" "<%= ctx[:primary_resource_id] %>" {
+  mute_config_id    = "<%= ctx[:vars]['config_id'] %>"
+  organization = "<%= ctx[:test_env_vars]['org_id'] %>"
+  location     = "global"
+  description  = "My custom Cloud Security Command Center Finding Organization mute Configuration"
+  filter = "category = \"OPEN_FIREWALL\" AND state = \"ACTIVE\""
```

Comment:
This value looks invalid- not sure it's it, I noticed that `=` is the separator here and not `:` like was used in V1: https://github.com/hashicorp/terraform-provider-google-beta/blob/main/google-beta/services/securitycenter/resource_scc_mute_config_generated_test.go#L65

Reply:
Hmm it should work just fine with `=`. In fact I think`:` is not necessarily a correct separator for us:
https://cloud.google.com/security-command-center/docs/how-to-api-filter-notifications#create-filter